src/styles/hc-2018/base.styl

```diff
@@ -1,4 +1,5 @@
 @import 'grid.styl';
```

Comment:
Would a more appropriate name for this be `all.styl`, since it contains all the styles needed for HC 2018?

Reply:
Good point

Reply:
The name is now `all-stylesheets.styl` for added disambiguation